runtimes/eden/src/xcm_config.rs

```diff
@@ -190,6 +191,8 @@ impl pallet_xcm::Config for Runtime {
 	type WeightInfo = crate::weights::pallet_xcm::WeightInfo<Runtime>;
 	#[cfg(feature = "runtime-benchmarks")]
 	type ReachableDest = ReachableDest;
+	type MaxRemoteLockConsumers = ConstU32<0>;
+	type RemoteLockConsumerIdentifier = ();
```

Comment:
Can we double check these

Reply:
https://github.com/paritytech/polkadot/pull/6947

https://github.com/paritytech/polkadot/pull/6947#issuecomment-1527287885

This is used in all applications and we don't use it either